src/main/resources/openapi.json

```diff
@@ -1,286 +1,935 @@
+                    "name": "partitionid",
+                    "in": "path",
+                    "description": "ID of the partition to which you want to send records.",
+                    "required": true,
+                    "schema": {
+                        "type": "string"
+                    }
+                }
+            ]
+        },
+        "/consumers/{groupid}/instances/{name}/positions": {
+            "post": {
+                "tags": [
+                    "Consumers"
+                ],
+                "description": "Configures a subscribed consumer to fetch offsets from a particular offset ID the next time it fetches a set of records from a given topic partition. This overrides the default fetch behavior for consumers. You can specify one or more topic partitions.",
+                "operationId": "seek",
+                "requestBody": {
+                    "description": "List of partition offsets from which the subscribed consumer will next fetch records.",
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "$ref": "#/components/schemas/Offsets"
+                            }
+                        }
+                    },
+                    "required": true
+                },
+                "responses": {
+                    "204": {
+                        "description": "No content."
+                    },
+                    "404": {
+                        "description": "The specified consumer instance was not found."
+                    }
+                }
+            },
+            "parameters": [
+                {
+                    "name": "groupid",
+                    "in": "path",
+                    "description": "ID of the consumer group to which the consumer belongs.",
+                    "required": true,
+                    "schema": {
+                        "type": "string"
+                    }
+                },
+                {
+                    "name": "name",
+                    "in": "path",
+                    "description": "Name of the subscribed consumer.",
+                    "required": true,
+                    "schema": {
+                        "type": "string"
+                    }
+                }
+            ]
+        },
+        "/consumers/{groupid}/instances/{name}/offsets": {
+            "post": {
+                "tags": [
+                    "Consumers"
+                ],
+                "description": "Commits a list of consumer offsets to the consumer offsets commit log. To commit offsets for all records fetched by the consumer, leave the request body empty.",
+                "operationId": "commit",
+                "requestBody": {
+                    "description": "List of consumer offsets to commit to the consumer offsets commit log. You can specify one or more topic partitions to commit offsets for.",
```

Comment:
If an empty body means "commit offsets for all records fetched by the consumer" then does a partial list of consumer offsets mean "commit just these offsets for these partitions, and don't commit anything for the other assigned partitions", or does it mean "commit these offsets for these partitions and for the others commit the offsets fetched by the consumer"?

Reply:
@ppatierno WDYT?